tests/TestMachineAction.py

```diff
@@ -63,14 +63,15 @@ def test_addMachineAction(machine_action_manager):
 
     # Check if adding multiple (the same) actions to first start actions work.
     machine_action_manager.addFirstStartAction(test_machine, "test_action")
-    machine_action_manager.addFirstStartAction(test_machine, "test_action")
-    assert machine_action_manager.getFirstStartActions(test_machine) == [test_action, test_action]
+    machine_action_manager.addFirstStartAction(test_machine, "test_action_2")
+
+    assert machine_action_manager.getFirstStartActions(test_machine) == [test_action, test_action_2]
```

Comment:
These tests do not test any more what the documentation says they would test. Please test:
* Adding multiple of the same actions.
* Inserting an action in any other location than the last location.